builder/googlecompute/config.go

```diff
@@ -443,16 +445,24 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 		}
 	}
 
+	if c.ImageArchitecture == "" {
+		c.ImageArchitecture = "x86_64"
```

Comment:
I wonder if this should default to x86_64, we need to double check regarding the values for `Image.Insert` in the API docs (doesn't seem to be indicated at first glance), but I wonder if we shouldn't leave it empty be default.
My rationale is that if the origin image's architecture setting is forwarded (from your description it doesn't seem to be, but I'd like to confirm), we'll break templates that may exist already.

Reply:
I think by default when this setting is nil or set to a blank string the GCP API defaults it to X86_64, only if this setting is explicitly set to a different string value would it change the templating? This is at least what I've seen in my local testing of building images.